### PR TITLE
Sourcery Pods and Mocking APIService Test Cases

### PR DESCRIPTION
**Pull request by rayandrie**

### CODE REVIEW

File: Example/Tests/Cases/Services/APIServiceTests.swift [hunk 1]
```diff
@@ -0,0 +1,279 @@
...
+        mockAPIService.auth(
+            userID: mock.userID,
+            issuer: mock.issuer,
+            event: mock.event,
+            code: mock.code,
+            method: mock.method,
+            pubKey: mock.pubKey,
+            timestamp: mock.timestamp
+        ) { _ in }
+
```
Comment (albertpurnama) [inline Example/Tests/Cases/Services/APIServiceTests.swift]: this is essentially mocking the thing that we're supposed to test?

File: Source/Service/Models/NotificationCredential.swift [hunk 2]
```diff
@@ -7,7 +7,7 @@
 
 import UIKit
 
-public class CotterNotificationCredential: Codable {
+public struct CotterNotificationCredential: Codable {
```
Comment (rayandrie) [inline Source/Service/Models/NotificationCredential.swift:10]: I made this change so that it matches all the other Model types, is fine right?

Comment (albertpurnama) [reply Source/Service/Models/NotificationCredential.swift:10]: yep, that's fine

**Review by albertpurnama — APPROVED** [review]
LGTM, need to add more test cases, but let's merge this first

**Pull request merged**